### PR TITLE
Add a "how to" for combining recordings 

### LINKED ISSUE #2620: Issue with reading intan file
Hi I am trying to read the intan .rhs file as given in the code below, but I get the following errors. Could you help whats going wrong here? 
Screenshot 2024-03-22 174436
Screenshot 2024-03-22 174520
Screenshot 2024-03-22 174655

### PR DESCRIPTION
**Pull request by zm711**
Fixes #2620.

@h-mayorquin just getting the ball rolling. Did we want to include aggregate channels at the bottom of this tutorial? 

I need to update the `repr` I was just trying to get the gist, but I'll try to do this with a real file so I can get the real repr values shortly :)

### CODE REVIEW

File: doc/how_to/combine_recordings.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    IntanRecordingExtractor 64 Channels - 30.0kHz - 1 segments - 1,800,000 samples - 60 seconds - uint16 dtype - xx
+    """
+
+    concatenated_recording = si.concatenate_recordings([intan_rec_one, intan_rec_two])
+
+    print(concatenated_recording)
+
+    """
+    ConcatenatedRecordingExtractor 64 Channels - 30.0kHz - 1 segments - 3,600,000 samples - 120 seconds - uint16 dtype -xx
+    """
```
Comment (h-mayorquin) [inline doc/how_to/combine_recordings.rst:49]: Could be a good idea to print out the two and point out the samples add up?

File: doc/how_to/combine_recordings.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
+How to Combine Recordings in SpikeInterface
+===========================================
+
+In this tutorial we will walk through combining multiple recording objects. This can happen when the probe is not
+moved within the tissue, but an overall recording session is split into several subsessions (e.g. pre-stim, stim, post-stim).
+Thus one may wish to combine these individual recording objects into one recording object before sorting.
+
+**Why Combine?**
+
+Spike sorters seek to sort spikes within a recording into groups of units. Thus if multiple :code:`Recording` objects have the
```
Comment (h-mayorquin) [inline doc/how_to/combine_recordings.rst:10]: Nice section. It was useful for me. I will add a concise answer at the beginning and then elaborate in the way that you do.

Something like:
Why combine?
Combining your data into a single recording allows to have consistent labels (.e.g unit_ids) across the whole recording.

What do you think?

File: doc/how_to/combine_recordings.rst (hunk 1 above)
Comment (h-mayorquin) [inline doc/how_to/combine_recordings.rst:49]: Maybe we can print the two and point out the samples add up?

File: doc/how_to/combine_recordings.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    recording = si.concatenate_recordings(list_of_recordings)
+
+
+Append Recordings
+^^^^^^^^^^^^^^^^^
+
+If you wish to keep each recording as a separate segment identity (e.g. if doing baseline, stim, poststim) you can use
+:code:`append` instead of :code:`concatenate`.
+
+#TODO Why would we really want to do this @everyone
```
Comment (h-mayorquin) [inline doc/how_to/combine_recordings.rst:69]: I wonder why as well.

**Review by h-mayorquin — COMMENTED** [review]
It would be nice to add the channels thing. But we should get this to the documentation and then do the channels in another PR just so we don't delay this unecessarily.

File: doc/how_to/combine_recordings.rst (hunk 1 above)
Comment (zm711) [reply doc/how_to/combine_recordings.rst:49]: Yep will do :)

File: doc/how_to/combine_recordings.rst (hunk 2 above)
Comment (zm711) [reply doc/how_to/combine_recordings.rst:10]: Sounds good. Will do.

File: doc/how_to/combine_recordings.rst (hunk 1 above)
Comment (zm711) [reply doc/how_to/combine_recordings.rst:49]: Yep. I needed to doublecheck the class titles as well to make sure I'm displaying the actual repr :)

File: doc/how_to/combine_recordings.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
+How to Combine Recordings in SpikeInterface
+===========================================
+
+In this tutorial we will walk through combining multiple recording objects. This can happen when the probe is not
```
[the remark this answers is not in the document]
Comment (zm711) [reply doc/how_to/combine_recordings.rst:4]: I'll try to fix. Don't want to be confusing.

[the remark this answers is not in the document]
Comment (zm711) [reply doc/how_to/combine_recordings.rst]: Tried to tweak this. Let me know what you think!

File: doc/how_to/combine_recordings.rst [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+    """
+    IntanRecordingExtractor: 64 Channels - 30.0kHz - 1 segments - 1,800,000 samples
+                             60.00s (1.00 minutes) uint16 dtype - 219.73 MiB
+    """
+
+    print(intan_rec_two)
+
+    """
+    IntanRecordingExtractor: 64 Channels - 30.0Khz - 1 segments - 1,800,000 samples
+                             60.00s (1.00 minutes) - unin16 dtype - 219.73 MiB
```
Comment (zm711) [inline doc/how_to/combine_recordings.rst:52]:
```suggestion
                             60.00s (1.00 minutes) - uin16 dtype - 219.73 MiB
```

File: doc/how_to/combine_recordings.rst [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+.. code-block:: python
+
+    recording = si.append_recordings([intan_rec_one, intan_rec_two])
+
+    print(recording)
+
+    """
+    AppendSegmentRecording: 64 Channels - 30.0khz - 2 segments - 3,600,000 samples
+                            120.00s (2.00 minutes) - unint16 dtype - 439.47 MiB
```
Comment (zm711) [inline doc/how_to/combine_recordings.rst:100]:
```suggestion
                            120.00s (2.00 minutes) - uint16 dtype - 439.47 MiB
```

**Review by zm711 — COMMENTED** [review]
Need to fix these. Always my reflex...

**Comment by zm711** [comment]
@h-mayorquin I'm ready for another read-through when you have time :)

**Review by h-mayorquin — APPROVED** [review]
This looks good to me. Let's see if @alejoe91 or @samuelgarcia have something to say.

Thanks a lot @zm711

**Pull request merged**